presto-main/src/main/java/com/facebook/presto/operator/DriverFactory.java

```diff
@@ -97,14 +95,14 @@ public synchronized Driver createDriver(DriverContext driverContext)
         return new Driver(driverContext, operators.build());
     }
 
-    @Override
     public synchronized void noMoreDriver()
     {
-        if (!closed) {
```

Comment:
Unrelated change?

Reply:
I moved it to `Rename DriverFactory.close to noMoreDrivers`. It's still an unrelated change. But this file is touched in that commit. And I don't think something this minor need a stand-alone commit.